connector/connector.go

```diff
@@ -641,3 +628,7 @@ func logStabilityLevel(logger *zap.Logger, sl component.StabilityLevel) {
 		logger.Info(sl.LogMessage())
 	}
 }
+
+func errDataTypes(id component.ID, from, to component.DataType) error {
```

Comment:
Based on https://github.com/open-telemetry/opentelemetry-collector/blob/main/component/component.go#L29 I think this should be:

```golang
	return fmt.Errorf("connector %q cannot connect from %s to %s: %w", id, from, to, component. ErrDataTypeIsNotSupported)
```

Reply:
Thanks, I've updated this